frontend/src/redux/actions/actions.tsx

```diff
@@ -199,6 +216,19 @@ export function addDetailsViewHeaderActionsProcessor(
   return { type: UI_DETAILS_VIEW_ADD_HEADER_ACTIONS_PROCESSOR, action: headerActionsProcessor };
 }
 
+export function addResourceTableColumnsProcessor(
+  tableProcessor: TableColumnsProcessor | TableColumnsProcessor['processor']
+) {
+  let tableColsProcessor = tableProcessor as TableColumnsProcessor;
+  if (tableColsProcessor.id === undefined && typeof tableProcessor === 'function') {
+    tableColsProcessor = {
+      id: '',
```

Comment:
Why is this an empty string, and in the other place it's generated?

Reply:
This is just because I want to do the generation in the reducer, not here. So we have it in one place. Same for the header actions processor.